### PR TITLE
SQ-321/Cleanup unused resources and code

### PR DESCRIPTION
**Pull request by rock3r**
This PR does a pass to remove all the leftover references to contest, proximity and location.

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -17,35 +17,34 @@ ext {
...
-                    firebaseMessaging   : "com.google.firebase:firebase-messaging:${playServicesVersion}",
-                    firebaseStorage     : "com.google.firebase:firebase-storage:${playServicesVersion}",
-                    firebaseUiStorage   : 'com.firebaseui:firebase-ui-storage:3.0.0',
-                    glide               : "com.github.bumptech.glide:glide:${glideVersion}",
-                    glideCompiler       : "com.github.bumptech.glide:compiler:${glideVersion}",
-                    glideOkHttp3        : "com.github.bumptech.glide:okhttp3-integration:${glideVersion}",
-                    jodaTimeAndroid     : 'net.danlew:android.joda:2.9.9',
-                    kotlin              : "org.jetbrains.kotlin:kotlin-stdlib-jre7:${kotlinVersion}",
-                    playServicesAuth    : "com.google.android.gms:play-services-auth:${playServicesVersion}",
-                    playServicesLocation: "com.google.android.gms:play-services-location:${playServicesVersion}",
```
Comment (rock3r) [inline dependencies.gradle:39]: This is the only line removed, the rest is just reformatted

File: app/build.gradle [hunk 2]
```diff
@@ -95,7 +95,7 @@ android {
 
     lintOptions {
         lintConfig teamPropsFile('static-analysis/lint-config.xml')
-        abortOnError false
+        abortOnError true
```
Comment (rock3r) [inline app/build.gradle:98]: We want the CI to go red if lint has errors.

Comment (fourlastor) [reply app/build.gradle:98]: fair enough but isn't there a way to go red after the errors looking at the report (this is good enough for now)

File: app/src/main/res/xml/remote_config_defaults.xml [hunk 3]
```diff
@@ -1,11 +1,2 @@
...
-  <entry>
-    <key>proximity_enabled</key>
-    <value>false</value>
-  </entry>
-  <entry>
-    <key>contest_goal</key>
-    <value>11</value>
-  </entry>
-</defaultsMap>
+<defaultsMap />
```
Comment (fourlastor) [inline app/src/main/res/xml/remote_config_defaults.xml:2]: do we need an empty map?

**Review by fourlastor — APPROVED** [review]
One question and good to 🚢 for me

File: app/src/main/res/xml/remote_config_defaults.xml (hunk 3 above)
Comment (rock3r) [reply app/src/main/res/xml/remote_config_defaults.xml:2]: It's just empty for now really, will be restored (albeit not for the London 2017 milestone). I don't want to remove it yet as we'll need it "soon"

File: app/build.gradle (hunk 2 above)
Comment (rock3r) [reply app/build.gradle:98]: We don't have a plugin to parse the lint outputs and evaluate thresholds unfortunately

**Pull request merged**